it/lessons/basics/basics.md

```diff
@@ -210,7 +210,7 @@ iex> 2 <= 3
 true
 ```
 
-Per una comparazione rigorosa (_strict_) tra interi e numeri in virogla mobile, usa `===`:
+Per una comparazione rigorosa (_strict_) tra interi e numeri in virgola mobile, usa `===`:
```

Comment:
I'd suggest the two following translations to "interi e numeri in virgola mobile":
1. "numeri interi ed in virgola mobile"
2. "numeri interi e numeri in virgola mobile"